specials/SPC30_Matias_IslandInfiltration.Malden/timer.sqf

```diff
@@ -0,0 +1,8 @@
+private "_time";
+_time = _this select 0;
+
+while {_time > 0} do {
+_time = _time - 1;  
+hintSilent format["Time Left: \n %1", [((_time)/60)+.01,"HH:MM"] call BIS_fnc_timetostring];	
```

Comment:
```suggestion
hintSilent format["Time Left: \n %1", [(_time/60)+.01,"HH:MM"] call BIS_fnc_timetostring];	
```
Why does this have `+0.1`?

Reply:
Suggestion's been commited. I don't really know why it has +0.1, I found the code online and it worked like a charm

Reply:
You can leave it then, just wondered